app/models/google_calendar/google_event.rb

```diff
@@ -13,8 +13,12 @@ def delete(event_id)
       GoogleCalendar::EventRemover.new(credentials).delete(event_id)
     end
 
-    def list_events(time_interval)
-      GoogleCalendar::EventFinder.new(credentials, user_email).list_events(time_interval)
+    def all(time_interval)
+      GoogleCalendar::EventFinder.new(credentials, user_email).all_events(time_interval)
+    end
+
+    def events_from(time_interval, conference_room_ids)
+      GoogleCalendar::EventFinder.new(credentials, user_email).events_from(time_interval, conference_room_ids)
```

Comment:
Looks like a good candidate to be extracted, cached and use as a service object for GoogleEvent.